images/cdh_service/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
We should call this file's directory `cloudera_service` because `CDH` is itself a parcel (which contains all its services). In this case, we're dealing with the installation of additional Cloudera services that didn't make it into CDH.

Reply:
Done.